index.html

```diff
@@ -135,12 +148,16 @@ <h3>CANOPY</h3>
                     <li>Javascript</li>
                 </ul>
                 <button class="See-project-button" type="button">See Project</button>
+                </div>
+
             </div>
+
         </div>
     </section>
     <!--about mew-->
     <section class="about-me-wrap">
-        <h2>About Myself</h2>
+        <div class="about-description-container">
+                    <h2>About Myself</h2>
```

Comment:
- [ ] Kindly, make sure the bottom margin matches the Figma design.

Your design:
![image](https://user-images.githubusercontent.com/86069740/236355297-905231d3-38cc-48e3-a0fa-172d83885586.png)

Figma design:
![image](https://user-images.githubusercontent.com/86069740/236355345-dab8e45f-a757-4dc2-b79a-252452e6af07.png)